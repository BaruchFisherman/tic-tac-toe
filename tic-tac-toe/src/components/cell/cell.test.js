import React from 'react';
import Cell from '.';
import { noop } from 'lodash';
import renderer from 'react-test-renderer';

describe('Cell', () => {
    it('Cell should render correctly when an X is there', () => {
        const component = renderer.create(
            <Cell 
                onClick={noop}
                content='X'
                color='red'
            />
        );

        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});