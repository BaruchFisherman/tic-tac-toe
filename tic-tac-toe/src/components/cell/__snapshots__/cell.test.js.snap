// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cell Cell should render correctly when an X is there 1`] = `
<div
  className="cell"
  onClick={[Function]}
  style={
    Object {
      "color": "red",
    }
  }
>
  X
</div>
`;
